src/azul/service/hca_response_v5.py

```diff
@@ -88,20 +95,23 @@ def for_bucket(cls, bucket: JSON) -> 'FileTypeSummary':
         self.totalSize = int(bucket['size_by_type']['value'])  # Casting to integer since ES returns a double
         self.matrixCellCount = int(bucket['matrix_cell_count_by_type']['value'])
         self.fileType = bucket['key']
+        self.format = bucket['key']
         return self
 
     @classmethod
     def for_aggregate(cls, aggregate_file: JSON) -> 'FileTypeSummary':
         self = cls()
         self.count = aggregate_file['count']
-        self.source = aggregate_file['source']
+        self.source = aggregate_file['file_source']
+        self.fileSource = aggregate_file['file_source']
```

Comment:
FIXME missing

and don't duplicate the initialization (however small) but instead create the backwards compatible entry like so

self.source = self.fileSource

and likewise for dictionaries.